tools/rc/usResourceCompiler.cpp

```diff
@@ -377,6 +373,58 @@ static int us_zip_writer_add_from_zip_reader(mz_zip_archive *pZip, mz_zip_archiv
   return us_zip_writer_add_dir_entries(pZip, archiveName, archived_dirs);
 }
 
+struct Custom_Arg : public option::Arg
+{
+  static void printError(const char* msg1, const option::Option& opt, const char* msg2)
+  {
+    std::cerr << "ERROR: " << msg1 << opt.name << msg2 << std::endl;
+  }
+  
+  static option::ArgStatus NonEmpty(const option::Option& option, bool msg)
+  {
+    if (option.arg != 0 && option.arg[0] != 0)
+      return option::ARG_OK;
+    
+    if (msg) printError("Option '", option, "' requires a non-empty argument\n");
+    return option::ARG_ILLEGAL;
+  }
+  
+  static option::ArgStatus Numeric(const option::Option& option, bool msg)
+  {
+    char* endptr = 0;
+    if (option.arg != 0 && strtol(option.arg, &endptr, 10)){};
+    if (endptr != option.arg && *endptr == 0)
+      return option::ARG_OK;
+    
+    if (msg) printError("Option '", option, "' requires a numeric argument\n");
+    return option::ARG_ILLEGAL;
+  }
+};
+
+#define US_PROG_NAME "usResourceCompiler"
```

Comment:
a static variable cannot be inline expanded like I want to in the usage[] at line#416